ui/src/components/EmailForm.vue

```diff
@@ -174,9 +315,33 @@ export default {
         cc: [],
         bcc: [],
         managerName: "", // TODO: use this field in the composed email later
```

Comment:
Are these TODOs resolved or should they stay in the code? If the latter, create an issue so we get back to them.

Reply:
This piece of code might be needed for the Issue-550. [https://github.com/corpus-christi/corpus-christi/issues/550](url), the issue is updated now